process/block/preprocess/transactions.go

```diff
@@ -1276,6 +1277,16 @@ func (txs *transactions) ProcessMiniBlock(
 
 		processedTxHashes = append(processedTxHashes, miniBlockTxHashes[index])
 
+		if txs.flagOptimizeGasUsedInCrossMiniBlocks.IsSet() {
+			//TODO: Remove the commented code if the meta-chain revert problem is fixed
```

Comment:
TODO still relevant?

Reply:
If the system test will pass the TODO and the commented code will be removed